Ip/Internal/Repository/assets/ipRepositoryAll.js

```diff
@@ -124,7 +198,6 @@
                 $this.find('.ipsBrowserSearch .fa-search').removeClass('fa-search').addClass('fa-times');
             } else {
                 // show all files if term doesn't exist
-                $files.removeClass('hidden');
```

Comment:
Why this is removed?

Reply:
I had some custom code here, removed too much by accident